meetings/2020_12_15/README.md

```diff
@@ -0,0 +1,27 @@
+# ROS 2 Security Working Group Meeting Minutes
+24 Nov 2020
```

Comment:
I don't think this is the right date.

Reply:
Updated with deb2fe866d6a1b873866901299fb43f0c0a2b341